app/models/portfolio_stock.rb

```diff
@@ -4,6 +4,16 @@ class PortfolioStock < ApplicationRecord
   belongs_to :portfolio
   belongs_to :stock
 
+  def change_amount
+    current_price = stock.current_price
+    normalized_purchase_price = purchase_price > 1000 ? purchase_price / 100.0 : purchase_price
```

Comment:
I'm confused here.  `purchase_price` is stored in the db and is a decimal right?

Reply:
<img width="1279" height="670" alt="Screenshot 2025-09-26 at 9 33 16 AM" src="https://github.com/user-attachments/assets/e5a7221b-9c6c-46ff-ad7e-4c39560e964a" />